tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -95,6 +101,18 @@ Object.Dai {
 			id	1
 		}
 	}
+	ALH."514" {
```

Comment:
@bardliao we need to start thinking about possible variations of the hardware layout. I think we should really create an 'amplifier' reference file and set the actual link/PDI number with macros, as we did for topology1.

Reply:
> @bardliao we need to start thinking about possible variations of the hardware layout. I think we should really create an 'amplifier' reference file and set the actual link/PDI number with macros, as we did for topology1.

Updated

Reply:
The ALH index is not used by driver. link_id will be provided by callback function.